integrationExamples/gpt/ias-multiple-tags.html

```diff
@@ -0,0 +1,196 @@
+<html>
+<head>
+
+<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
+<script>
+  var googletag = googletag || {};
+  googletag.cmd = googletag.cmd || [];
+</script>
+
+<script>
+googletag.cmd.push(function() {
+	googletag.pubads().disableInitialLoad();
+});
+</script>
+
+<script>
+  googletag.cmd.push(function() {
+    googletag.defineSlot('/57514611/jwack/pub_opt/top_banner', [[970, 250], [728, 90], [1000, 90]], 'div-gpt-ad-1492761750660-0').addService(googletag.pubads());
+    googletag.defineSlot('/57514611/news.com', [[300, 250], [250, 250]], 'div-gpt-ad-1492761750660-1').addService(googletag.pubads());
+    googletag.pubads().enableSingleRequest();
+    googletag.enableServices();
+  });
+</script>
+
+<script>
+var PREBID_TIMEOUT = 700;
+
+var adUnits = [{
+	code: 'div-gpt-ad-1492761750660-0',
+    sizes: [[970, 250], [728, 90], [1000, 90]],
+    bids: [{
+        bidder: 'ias',
+        params: {
+           pubId: '8713',
+           adUnitPath: '/57514611/jwack/pub_opt/top_banner'
```

Comment:
I think it's fine if we just keep the test page to our forked repo: the ad creative here is directly from a client - not sure if we want to commit it to a open source repo.